source/tyr/tyr/resources.py

```diff
@@ -1081,3 +1081,18 @@ def post(self, ac_instance_name):
         _, job = import_autocomplete([filename], instance)
         job = models.db.session.merge(job) #reatache the object
         return marshal({'job': job}, one_job_fields), 200
+
+
+class MigrateFromPoiToOsm(flask_restful.Resource):
+    def delete(self, instance_name):
```

Comment:
minor consistency: in Tartare, POST method is used https://github.com/CanalTP/tartare/blob/master/documentation/ressources/open_api.yaml#L183
But @kinnou02 is right, it's a delete, so...